Source/Common/PropertyValueSpecification.Partial.cs

```diff
@@ -9,6 +8,13 @@ namespace Schema.NET;
 /// </summary>
 public partial class PropertyValueSpecification
 {
+    private const string MaxLengthPropertyName = "maxlength=";
+    private const string MinLengthPropertyName = "minlength=";
+    private const string NamePropertyName = "name=";
+    private const string PatternPropertyName = "pattern=";
+    private const string RequiredPropertyName = "required";
+    private const char Space = ' ';
```

Comment:
The others above seem reasonable but bringing the space out seems a little strange.

Reply:
Is that because its a `char`? Do you not normally make those constants?

Reply:
Yeah - like I think where constants shine are when:
- You have multiple instances of it (this doesn't always matter but is usually a good reason)
- It isn't immediately obvious _why_ the value is what is it (this is typically the main reason)
- The value is significant (eg. a long value that doesn't fit inline very well to the code)

Admittedly I could say _all_ of the constants here don't meet these requirements but also it can be an amount of personal preference too.

Like, nothing seems too "magic" about the values to need to be brought up to constants but nothing seems too bad about doing it.

Where my thoughts get extra complicated on it is _where_ the constants are. For example, maybe these as constants would make more sense inline right where they're used rather than on the class itself.

To be clear though, if you prefer it this way, happy to go that direction even if it is atypical to what I might do. 🙂

Reply:
OK, it turns out that I just made a `char` a constant in some code I'm working on. In this case, my constants were like:

```csharp
private const char PathEndSpecialCharacter = '$';
private const char WildcardSpecialCharacter = '*';
```

They described the purpose or perhaps the meaning of the value rather than the value.

Reply:
Renamed `Space` to `SpaceDelimeter`.

Also updated to the latest .NET 7 preview release.